code-challenges.js

```diff
@@ -58,12 +127,40 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+// Function
+const lastLocation = (number) => {
+    return numberOfConnections.lastIndexOf(number)
+}
+
+console.log(lastLocation(givenValue1));
+// Output: 7
+console.log(lastLocation(givenValue2));
+// Output: 8
+
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Pseudo code:
+// Input: one numerical array
+// Output: the given array sorted by largest to smallest
+// Process:
+// Create a function called sortedArray
+// Takes in one array
+// Sort the given array (possible tool: .sort() mutator with '(a, b) => b-a' algorithm)
+// Return the sorted array
 
 const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
 // Expected output: [82, 80, 79, 77, 76, 73, 72]
 
 const sanDiegoWinterTemperatures = [59, 68, 62, 59, 66, 67, 66]
 // Expected output: [68, 67, 66, 66, 62, 59, 59]
+
+// Function
+const sortedArray = (array) => {
+    array.sort((a, b) => b - a)
+    return array
+}
```

Comment:
Nice research! No need to return array here, you can just return array.sort(....)!